Marlin/Configuration.h

```diff
@@ -1564,4 +1564,59 @@
   //#define FILAMENT_LCD_DISPLAY
 #endif
 
+//===========================================================================
+//============================ I2C Encoder Settings =========================
+//===========================================================================
+
```

Comment:
Great idea. Probably include links to the wiki and the github for the encoder board.